doc/source/user_guide_embedding/globals.rst

```diff
@@ -24,3 +23,12 @@ following code:
    # The following line extracts the global API entry points and merges them into your global
    # Python global variables.
    globals().update(global_variables(app))
+
+
+By default enums, such as `SelectionTypeEnum` or `LoadDefineBy` are not included in these
```

Comment:
Because they number in the thousands, by default, enums, such as `SelectionTypeEnum` or `LoadDefineBy` are not included in these global variables. To include enums, set the second argument of `global_variables`` to True.

Reply:
@sbradrick I've reworded it to something like you suggest [here](https://github.com/ansys/pymechanical/pull/412)